### PR TITLE
chore(deps): update

### PR DESCRIPTION
**Pull request by evilebottnawi**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [x] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### Motivation / Use-Case

update deps

### Breaking Changes

Should be no

### Additional Info

No

/cc @hiroppy 

What we need to do in next PRs:
- Fix all ` ConcurrentCompilationError: You ran Webpack twice. Each instance only supports a single concurrent compilation at a time.` errors in tests
- Fix `.skip` test in `Compress.test.js`
- Increase coverage and more tests

Can you take care?

### CODE REVIEW

File: test/Proxy.test.js [hunk 1]
```diff
@@ -69,9 +69,9 @@ function startProxyServers() {
 
 describe('Proxy', () => {
   // issue: https://github.com/webpack/webpack-dev-server/pull/1633#issuecomment-463938675
-  if (shouldSkipTestSuite()) {
-    return;
-  }
+  // if (shouldSkipTestSuite()) {
+  //  return;
+  // }
```
Comment (evilebottnawi) [inline test/Proxy.test.js]: Let's see on issue status with updated `http-proxy-middleware`

Comment (hiroppy) [reply test/Proxy.test.js]: it seems good with Node11 🎉 https://travis-ci.org/webpack/webpack-dev-server/jobs/495415842

Comment (evilebottnawi) [reply test/Proxy.test.js]: @evilebottnawi no :smile: https://github.com/webpack/webpack-dev-server/pull/1672

**Comment by hiroppy** [comment]
wait for #1672

**Comment by evilebottnawi** [comment]
Problems with proxy solved :+1: Feel free to feedback

**Pull request merged**